client/components/tinymce/index.jsx

```diff
@@ -153,9 +153,9 @@ PLUGINS.push( 'wpcom/mentions' );
 
 const CONTENT_CSS = [
 	window.app.tinymceWpSkin,
-	'//s1.wp.com/wp-includes/css/dashicons.css',
+	'//s1.wp.com/wp-includes/css/dashicons.css?v=20150727',
```

Comment:
This looks great - I wonder, though, if we should pull this in to a constant so that we don't need to worry about updating in many places in the future?
Though I'll admit, I wouldn't know how to do that in jade templates without refreshing myself a little :P

Reply:
I'm not quite sure how to do that - this value is used across `.jade`, `.jsx` as well as `.scss` files.

Reply:
Yeah that crossed my mind - could be more hassle than it's really worth!

Reply:
Dw, https://github.com/Automattic/wp-calypso/pull/18297 should be addressing that

Reply:
Apologies if I'm missing something but I'm not quite sure how #18297 addresses that. Do you mind explaining a bit more?